examples/django/.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
We don't want `.idea` files to be checked in. Please ignore them.

Reply:
deleted the .idea directory.